apps/design/frontend/src/export_screen.tsx

```diff
@@ -238,10 +212,10 @@ export function ExportScreen(): JSX.Element | null {
         </P>
         <P>
           {isElectionPackageExportInProgress ? (
-            <LoadingButton>Exporting Election Package...</LoadingButton>
+            <LoadingButton>Exporting Zip File...</LoadingButton>
```

Comment:
I think I might prefer "Exporting Election Package & Ballots..." here

Reply:
I was thinking that much text might make for a very wide button, but since this button is for support users only it's probably fine